app/controllers/obituaries/recent_controller.rb

```diff
@@ -0,0 +1,4 @@
+class Obituaries::RecentController < ApplicationController
```

Comment:
I assume for every category we are going to create a separate controller for the index method in order to keep it ReSTful. It feels weird but I think it is right.

Reply:
I think its right too. Fat methods, skinny controllers.

Reply:
looks good